tests/test_parser.py

```diff
@@ -184,6 +184,24 @@ async def test_parsing_docx_type():
     assert results[0]["content"] is not None
 
 
+@pytest.mark.asyncio
+async def test_parsing_xlsx_type():
+    sample = "examples/inputs/sample.xlsx"
+    parser_type = "STATIC_PARSE"
+    results = parse(sample, parser_type)["segments"]
+    assert len(results) >= 1
+    assert results[0]["content"] is not None
```

Comment:
For structured data, lets keep them in `parquet` format for now?
Depending on usage one could convert the frame for downstream tasks?

Reply:
Sorry didn't get what you meant here either. Do you mean keep the files saved in `parquet` format?

Reply:
Yes, as a more efficient method of returning rows and columns; should we also consider adding the option to save and retrieve as `parquet`? So 2 options `str`/`parquet`?

Reply:
Shouldn't we use an Excel file because the test is for excel files?
We can and probably should extend support for reading parquet files as well. I can include that in this PR.

> Should we also consider adding the option to save and retrieve as parquet? So 2 options str/parquet?

I'm not sure whether support for file conversion should be part of Lexoid. It made sense in the case of conversion to PDF because many language models accept it. I'm not sure about this case.

Reply:
Conversion here is related to the passing output format for downstream tasks, which we want to do in a secure and efficient way.

Reply:
Issue tracked in #87